TLM/TLM/UI/SubTools/LaneArrows/LaneArrowTool.cs

```diff
@@ -179,23 +180,38 @@ private enum Trigger {
         /// </summary>
         /// <param name="numLanes">How many groups of buttons.</param>
         private void CreateLaneArrowsWindow(int numLanes) {
-            var parent = UIView.GetAView();
-            ToolWindow = (LaneArrowToolWindow)parent.AddUIComponent(typeof(LaneArrowToolWindow));
-            ToolWindow.SetOpacity(
-                U.UOpacityValue.FromOpacity(0.01f * GlobalConfig.Instance.Main.GuiOpacity));
+            // var parent = UIView.GetAView();
```

Comment:
Will that commented code be used in the future (next batch) or you forgot to remove it after refactoring?

Reply:
This was old code before i copied in the new code. Was not sure whether it will just become a drop-in replacement, and it did. Removed the commented code.